packages/firebase_database/firebase_database/pubspec.yaml

```diff
@@ -1,7 +1,7 @@
 name: firebase_database
 description: Flutter plugin for Firebase Database, a cloud-hosted NoSQL database
   with realtime data syncing across Android and iOS clients, and offline access.
-homepage: https://github.com/FirebaseExtended/flutterfire/tree/master/packages/firebase_database
+homepage: https://github.com/FirebaseExtended/flutterfire/tree/master/packages/firebase_database/firebase_database
 version: 3.1.6
```

Comment:
You need to bump the version and update the CHANGELOG.md describing this change, otherwise this won't be publishable into pub.dev.

Reply:
I thought we are not gonna publish it to pub.dev unless we have some actual changes to the functionality.  I have added the suggested change anyway.

Reply:
Thanks! In case that `3.1.6+1` is not accepted by the build script, feel free to move it to `3.1.7`.

Reply:
Sure.